File: Java/xunit-test-patterns/src/main/java/_03_parameterized_tests/Person.java
package _03_parameterized_tests;

public class Person {
    public static final int ADULT_AGE = 18;
    private int age;

    public Person(int age) {
        this.age = age;
    }

    public boolean isAdult() {
        return age >= 18;

    }
}

File: CSharp/XUnitTestPatterns/XUnitTestPatterns/_03_ParameterizedTests/Person.cs
namespace XUnitTestPatterns._03_ParameterizedTests
{
  public class Person
  {
    public const int AdultAge = 18;
    private readonly int _age;

    public Person(int age)
    {
      _age = age;
    }

    public bool IsAdult()
    {
      return _age >= AdultAge;
    }
  }
}